packages/twenty-front/src/modules/settings/hooks/useExpandedAnimation.ts

```diff
@@ -21,16 +21,19 @@ const advancedSectionAnimationConfig = (
   isExpanded: boolean,
   dimension: AnimationDimension,
   measuredValue?: number,
```

Comment:
Maybe `measuredValue` can be renamed with something like `size` of type `number | 'fit-content'`, the in the hook `useExpandedAnimation` we can have another union called `mode` with type `'scroll-height' | 'fit-content'`

Reply:
agree